perllib/FixMyStreet/App/Controller/Moderate.pm

```diff
@@ -234,9 +223,9 @@ sub moderate_update : Chained('update') : PathPart('') : Args(0) {
     $c->forward('update_moderate_hide');
 
     my @types = grep $_,
```

Comment:
I feel like there ought to be a check in here that we've found the comment given it falls back to moderating the problem now.